Analysis/backgrounds_efake.tex

```diff
@@ -1,14 +1,22 @@
-An electron can be misidentified as a photon if the association of tracks or track seeds to the ECAL supercluster fails in the reconstruction step. 
+An electron can be misidentified as a photon if the association of
+tracks or track seeds to the ECAL supercluster fails in the
+reconstruction step. \PH{Grammar: fail}
 The production of a single \PW\ boson decaying to an electron and a neutrino is a high-rate process, and it mimicks the photon plus \met\ signature if the electron is misidentified.
 
 The rate  at which this misidentification occurs is $R_{\Pe} = (1 - \epsilon_{\Pe}^{\text{track}}) / \epsilon_{\Pe}^{\text{track}}$, where $\epsilon_{\Pe}^{\text{track}}$ is the tracking efficiency of electrons passing the photon identification criteria except for the electron veto.
-After making the reasonable assumption that the kinematic and other critical properties of the electron plus \met\ events are unaffected by the electron misidentification, we model the electron misidentification background by taking the electron proxy sample and scaling it by $R_{\Pe}$.
+After making the reasonable assumption that the kinematic and other
+critical properties of the electron plus \met\ events are unaffected
+by the electron misidentification, we model the electron
+misidentification background by taking the electron proxy sample and
+scaling it by $R_{\Pe}$.
+\PH{Move this paragraph to the end of this section. Here it is a bit disjoint}
```

Comment:
moved second sentence to the end of the section. and merged first one with the next paragraph.